lms/js_config_types.py

```diff
@@ -7,6 +7,12 @@
 from typing import NotRequired, TypedDict
 
 
+class AnnotationMetrics(TypedDict):
```

Comment:
Common struct for all of this, move to the top to be available for the rest.

This is the same as the old `AssignmentStats`